astrocut/cube_cut.py

```diff
@@ -720,8 +823,13 @@ def cube_cut(self, cube_file, coordinates, cutout_size,
         warnings.filterwarnings("ignore", category=wcs.FITSFixedWarning)
         with fits.open(cube_file, mode='denywrite', memmap=True) as cube:
 
+            # Get the first FFI file, which is used to generate the 
+            # primary header keyword values 
+            self.first_ffi = cube[2].data['FFI_FILE'][0]
+            self.last_ffi = cube[2].data['FFI_FILE'][-1]
```

Comment:
where do these attributes (`first_ffi` and `last_ffi`) get used?

Reply:
I made these calls when attempting to add header keywords for these specific files, but the filenames are too long so its been left as a nice-to-have TODO item. I'll remove these calls from this PR since it's now out of scope.